### PR TITLE
Formulae versioning of ReadTools

### PR DESCRIPTION
**Pull request by magicDGS**
Using the versioning syntax from Homebrew (see https://docs.brew.sh/Versions.html) to keep several formulae of ReadTools.
For keeping the latest version in the tap, and Aliases folder was added using the approach described in https://discourse.brew.sh/t/how-do-i-add-multiple-versions-to-a-formula-on-github/910/10

Changes:

* Rename readtools formulae to include the version number (including class name)
* New formulae for version 1.1
* Add `keg_only :versioned_formula` to version 1.0
* Alias for readtools latest version (1.1, should be updated in every release)

### CODE REVIEW

**Comment by magicDGS** [comment]
Can you have a look to this update, @robmaz? Let me know if I need to change something or if it makes sense this structure for our purposes...

File: Aliases/readtools [hunk 1]
```diff
@@ -0,0 +1 @@
+../readtools@1.1.rb
```
Comment (robmaz) [inline Aliases/readtools]: link

File: readtools@1.0.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-class Readtools < Formula
+class ReadtoolsAT10 < Formula
```
Comment (robmaz) [inline readtools@1.0.rb]: add 
keg_only :versioned_formula

File: Aliases/readtools (hunk 1 above)
Comment (magicDGS) [reply Aliases/readtools]: Done

File: readtools@1.0.rb (hunk 2 above)
Comment (magicDGS) [reply readtools@1.0.rb]: Done

**Comment by magicDGS** [comment]
Back to you @robmaz - I realized that I have merge access to the repo, I am not sure if that's what you want...

**Comment by robmaz** [comment]
I think it works now. I just had to add all the files to my master branch,
commit, and push.
That is also exactly what we tried yesterday, I believe, but probably we
were too hasty with
one of those steps.

Cheers
Rupert

2017-10-05 18:00 GMT+02:00 Daniel Gómez-Sánchez <notifications@github.com>:

**Comment by magicDGS** [comment]
Nice, thanks! I am filling in issues to have the repository working in a better way, with clear contribution guidelines and so on. If a bug like the one introduced here happens, maybe we can fill in an issue and like that discuss there the details for keeping track of the changes instead of pushing directly to master.

**Pull request merged**